src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp-troubleshooting.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: New Relic OTLP Troubleshooting
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+  - Troubleshoot
+metaDescription: Troubleshoot common OTLP ingest errors
+freshnessValidatedDate: never
+---
+
+New Relic has supported [native OTLP ingest](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) for several years. In the process of working through support cases that come up from time to time, we've learned about common issues users face. For some, the problem is easy to identify and fix. Others are deviously tricky, given that the internet is unreliable and there are many components (software, networking, hardware, etc) involved under the control of various parties (customers, New Relic, and public networking infrastructure outside the control of either). With so much complexity, configuration, and failure points, it can be difficult to determine which is at fault an how to best address.
```

Comment:
```suggestion
New Relic has supported [native OTLP ingest](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) for several years. In the process of working through support cases that come up from time to time, we've learned about common issues users face. For some, the problem is easy to identify and fix. Others are deviously tricky, given that the internet is unreliable and there are many components (software, networking, hardware, etc.) involved under the control of various parties, such as customers, New Relic, and public networking infrastructure outside the control of either. With so much complexity, configuration, and failure points, it can be difficult to determine which is at fault and how to best address.
```